backend/app/rabbitmq/heartbeat_listener_sync.py

```diff
@@ -62,7 +58,7 @@ def callback(ch, method, properties, body):
             processed_feed = await _process_incoming_v1_extractor_info(
                 extractor_name, extractor_out.id, process
             )
-            db["feeds"].insert_one(processed_feed)
+            processed_feed.insert()
```

Comment:
wonder if we need await here?

Reply:
good catch, added this and a few other missing awaits also